### PR TITLE
Add Python 3.9 wheel for Linux

### PR DESCRIPTION
**Pull request by jackgerrits**
- Windows is pending compatible vcpkg packages being available
- MacOS is pending the required dependencies to be available for 3.9 on conda forge

### CODE REVIEW

File: .github/workflows/build_python_wheels_macos.yml [hunk 1]
```diff
@@ -43,6 +44,7 @@ jobs:
         - { version: 3.6, include_dir_name: python3.6m/}
         - { version: 3.7, include_dir_name: python3.7m/}
```
Comment (lalo) [inline .github/workflows/build_python_wheels_macos.yml:45]: why do some end on m and others dont?

Comment (jackgerrits) [reply .github/workflows/build_python_wheels_macos.yml:45]: Not 100% sure but it changed over the course of the releases

**Comment by jackgerrits** [comment]
Looks like there are dependency issues with the conda activate

File: .github/workflows/build_python_wheels_macos.yml (hunk 1 above)
Comment (peterychang) [reply .github/workflows/build_python_wheels_macos.yml:45]: the `m` suffix indicates its built using a pymalloc allocator. In most cases i've seen its the same thing (3.6 symlinks to 3.6m), but technically the libraries are not guaranteed to be binary compatible

**Comment by peterychang** [comment]
LGTM

**Pull request merged**